platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/eeprom.py

```diff
@@ -64,6 +64,11 @@
 EEPROM_SYMLINK = "/var/run/platform/eeprom/syseeprom"
 EEPROM_STATUS = "/var/run/platform/eeprom/status"
 
+try:
+    _str_type = basestring
+except NameError:
+    _str_type = str
```

Comment:
All applications which import sonic-platform should now use the Python 3 API if it is available (as of a couple weeks ago). Therefore, you *should* be able stop building/installing a Python 2 sonic-platform package and you could then remove this check, as the string type will always be `str` with Python 3.

If you'd prefer to keep this for now and leave the remove of removal of Python 2 support for a separate PR, that's OK. Just let me know.

Reply:
ok, I will prepare a separate PR for this.

Reply:
Sure thing. I just wanted to prevent you from adding code if not necessary. We can merge this as-is and you can clean it up with the subsequent PR, once you're able to prove you no longer need to build/install a Python 2 package.

Reply:
Thanks for info!